### PR TITLE
project-schema.json: add environment.abatementCost

### LINKED ISSUE #416: Carbon efficiency (abatement cost)
# Background

This issue relates to the following CoST IDS elements proposed in the CoST IDS/OC4IDS review:

<details>

<summary> Carbon efficiency (Preparation) </summary>

## Data point title

**Module:** Climate finance
**Indicator:** Climate impact

### Disclosure format

> Disclose the cost per tonne of CO2 equivalent [value, currency].

### OC4IDS mapping

> Publish the cost in `environment.abatementCost`. If supporting documentation is available, publish in documents with `.documentType` set to 'abatementCostMethodology'.

</details>

# Proposal

Add the following fields to the `environment` object proposed in #393:

Path | Title | Description | Type (format) | Codelist
-- | -- | -- | -- | --
environment.abatementCost | Abatement cost | The lifetime abatement cost of the project, measured in cost per tonne of CO2 equivalent emissions. That is, the additional cost of reducing greenhouse gas emissions by one tonne compared to the most likely alternative scenario. | `Value` object |  

Add the following codes:

Codelist | Code | Title | Description
-- | -- | -- | --
documentType | abatementCostMethodology | Abatement cost methodology | The methodology used to calculate the project’s abatement cost.

# Example

```json
{
  "environment": {
    "abatementCost": {
      "amount": 12.29,
      "currency": "NZD"
    }
  },
  "documents": [
    {
      "id": "1",
      "documentType": "abatementCostMethodology",
      "url": "http://example.com/abatementCostMethodology.pdf"
    }
  ]
}
```

# Sources

* [What You Need to Know About Abatement Costs and Decarbonization - World Bank Climate Explainer Series](https://www.worldbank.org/en/news/feature/2023/04/20/what-you-need-to-know-about-abatement-costs-and-decarbonisation): Field name, title and description
* [Abatement cost methodology for assessing co-funding applications - Energy Efficiency and Conservation Authority, New Zealand](https://www.eeca.govt.nz/assets/EECA-Resources/Co-funding/GIDI-Files/GIDI-levelised-abatement-cost-methodology.pdf): Field description - concepts of additionality and lifetime abatement costs (page 5), example of abatement cost methodology (pages 18-22)
* [Green Infrastructure Finance: Framework Report - The World Bank](https://openknowledge.worldbank.org/server/api/core/bitstreams/a1cb7ad9-64c9-59c6-87b1-21df4e1a1974/content): Field description - concept of lifetime abatement cost (page 18)

cc @evelyndinora @mgraca-prado

### PR DESCRIPTION
**Pull request by odscjen**
**Related issues**

closes #416 

**Description**

**Merge checklist**

- [x] [Log your changes](https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html#logging-changes)

If there are changes to `project-schema.json` or `project-package-schema.json`:

- [x] Update the examples:
  - [ ] `docs/examples/example.json`
  - [ ] `docs/examples/blank.json`
- [x] Run `./manage.py pre-commit` to update `docs/_static/i8n.csv`

If you added a new definition to the schema, run `./manage.py pre-commit`.

If you added a new codelist:

  - [ ] Add an entry to `docs/reference/codelists.md`

### CODE REVIEW

File: docs/examples/example.json [hunk 1]
```diff
@@ -986,7 +986,11 @@
       "environment": {
         "goals": [
           "climateChangeMitigation"
-        ]
+        ],
+        "abatementCost": {
+          "amount": 50000,
```
Comment (duncandewhurst) [inline docs/examples/example.json]: Minor point, but the example abatement costs that I found when researching this field were much smaller than 50,000 USD.

```suggestion
          "amount": 12.29,
```

File: mapping/sustainability.yaml [hunk 2]
```diff
@@ -783,8 +783,23 @@
     module: Climate finance
     indicator: Climate impact
     disclosure format: Disclose the cost per tonne of CO2 equivalent \[value, currency\].
-    mapping: Publish the cost in `environment.abatementCost`. If supporting documentation is available, publish in documents with `.documentType` set to 'abatementCostMethodology'.
-    example: ''
+    mapping: Publish the cost in `environment.abatementCost`. If supporting documentation is available, publish in `documents` with `.documentType` set to 'abatementCostMethodology'.
```
Comment (duncandewhurst) [inline mapping/sustainability.yaml]: Copy edit and reference common operation.

```suggestion
    mapping: Map to `environment.abatementCost`. If supporting documentation is available, [add a project document](../common.md#add-a-project-document) and set `.documentType` to 'abatementCostMethodology'.
```

File: mapping/sustainability.yaml [hunk 3]
```diff
@@ -783,8 +783,23 @@
...
+    example: |-
+        {
+          "environment": {
+            "abatementCost": {
+              "amount": 50000,
```
Comment (duncandewhurst) [inline mapping/sustainability.yaml]: Per example in the issue.

```suggestion
              "amount": 12.29,
```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
Happy for you to request a review from James once you've resolved my comments and run `update-sustainability-docs`.

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**